PWGDQ/TableProducer/tableMaker.cxx

```diff
@@ -466,6 +466,12 @@ struct TableMaker {
       fullSkimming<gkEventFillMap, gkTrackFillMap, 0u>(collision, bcs, tracksBarrel, nullptr);
     }
   }
+  // Produce barrel only tables, with Q vector ----------------------------------------------------------------------------------------
+  void processBarrelOnlyWithQvector(MyEventsWithCent::iterator const& collision, aod::BCs const& bcs,
+                                    soa::Filtered<MyBarrelTracks> const& tracksBarrel)
+  {
+    fullSkimming<gkEventFillMap, gkTrackFillMap, 0u>(collision, bcs, tracksBarrel, nullptr);
+  }
```

Comment:
We need indeed to create a process function for the case when the Q vector is available and needs to be skimmed, however, I don't see the needed changes in the "fullSkimming" function. At the moment, this process function does not produce a data model where the Q vector table is added together with the rest of the data model

Reply:
I removed this process in the table-maker.